.tool-versions

```diff
@@ -0,0 +1 @@
+java temurin-21.0.3+9.0.LTS
```

Comment:
bruker asdf for å automatisk sette java versjon fra working directory. Håper det er greit å legge til denne 🙏

Reply:
Ukjent for meg, gir det ikke mening å legge til i .gitignore?

Reply:
Klarer ikke finne den sida igjen nå. Men når jeg begynte her var det en anbefaling på nais doc'en å bruke asdf for å håndtere forskjellige versjoner av binaries (ikke bare java, men node, kubectl, osv). Fordelen med å sjekke den inn er at om man bruker asdf for slikt så settes det automatisk når det jobbes med et repo.